src/components/Modal.tsx

```diff
@@ -15,7 +15,7 @@ const Message = styled(HeadingText)`
   text-align: center;
 `
 
-export interface CustomModalProps {
+export interface ModalProps {
```

Comment:
(optional) we try to name prop types just `Props`, but as this is not consistent yet over the project, just fine if you keep this here, too. But just fyi.

Reply:
thanks for letting me know, changed it.